runtime/src/bank.rs

```diff
@@ -6955,6 +6893,59 @@ impl TransactionProcessingCallback for Bank {
             LoadedProgramMatchCriteria::NoCriteria
         }
     }
+
+    // NOTE: must hold idempotent for the same set of arguments
+    /// Add a builtin program account
+    fn add_builtin_account(&self, name: &str, program_id: &Pubkey, must_replace: bool) {
```

Comment:
Except for a few tests, `must_replace` is always `false`. @Lichtso can this be removed, and assumed to be false always?

Reply:
I think we can remove that flag.

Reply:
> I think we can remove that flag.

I can make a separate PR for the flag removal. This way, such a change won't be missed in the moved code.